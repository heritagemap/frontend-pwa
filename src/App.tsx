import React from 'react';
import {
  BrowserRouter as Router,
  Switch,
  Route,
} from 'react-router-dom';

import Map from 'components/Map';
import DefaultMap from 'components/DefaultMap';
import Sidebar from 'components/Sidebar';
import MonumentPage from 'components/MonumentPage';

import './App.scss';

const App = () => (
  <Router>
    <main>
      <Switch>
        <Route path="/lat/:lat/lon/:lon/:id?">
          <Map />
          <Sidebar />
        </Route>

        <Route path="/:id">
          <MonumentPage />
        </Route>

        <Route exact path="/">
          <DefaultMap />
        </Route>
      </Switch>
    </main>
  </Router>
);

export default App;
