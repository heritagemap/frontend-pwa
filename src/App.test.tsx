import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

test('renders App', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
