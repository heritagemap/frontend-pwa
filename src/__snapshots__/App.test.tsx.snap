// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App 1`] = `
<div>
  <div
    class="map"
    id="map_container"
  />
</div>
`;
